### ISSUE TITLE
Conditional generation suggestions

### ISSUE THREAD

**Issue by olavdc-dl** [EperLuo/scDiffusion#17]
Hi @EperLuo, thanks a bunch for developing this cool project ! 

I've been testing the multi-conditional generation on a toy dataset with the following metadata : 

` AnnData object with n_obs × n_vars = 4000 × 58676
    obs: 'disease', 'organ' `

where disease has categories `['cancer', 'healthy']` and organ has categories`['heart', 'liver']`

I'm trying to generate data with the following characteristics: 

`{'disease' : 'healthy', 'organ' : 'heart'}` 

I'm following your `script_diffusion_multi-condi.ipynb` notebook as this is a very similar case study but I might have a questions regarding the multi-conditional generation: 

In the issue [#16 ](https://github.com/EperLuo/scDiffusion/issues/16) you remind that the AE (scSimilarity) should see the same cell types at inference than during training. Does that means that in `script_diffusion_multi-condi.ipynb` you use a AE which is finetuned on data with the condition (`mammary B` cell and `mammary T cell`) ? If yes, do you use the same AE in the training and in the inference ? If not, could you detail the generation procedure you recommend ? 

Thanks again and looking forward to your answer !

**Comment by EperLuo** [comment]
Hi! Thanks a lot for your interest. We didn't use mammary B cells during the fine-tune process of AE. Neither the AE nor the diffusion backbone has seen the mammary B cell at all (for both the training and inference process). Again, we recommend that you decode the unseen cell type together with those known cell types (which you used in the training process) to reduce the effect of normalization. And the unseen cell type shouldn't be the dominant cell in the decode batch (you may try different proportions to find the optimal setting).

**Comment by olavdc-dl** [comment]
Hi ! Thanks a lot for the quick and clear answer. 

I think I understand how to proceed with the sampling now. I believe we can close this issue. Thanks again !
